### ISSUE TITLE
Syntax highlighting

### ISSUE THREAD

**Issue by gldtn** [scottmckendry/cyberdream.nvim#137]
### Question or Suggestion

I just noticed while I was playing with italics that my syntax highlighting is different from the original cyberdream for some reason:

Any idea what might be overriding? Or what I can check?
![syntax](https://github.com/user-attachments/assets/30ad617c-02ff-4fe3-9f0c-93fe41d3c178)

Thanks!

**Comment by scottmckendry** [comment]
Hey @gldtn!

Can you be more specific about what is wrong? Nothing is jumping out at me from your screenshot.

**Comment by gldtn** [comment]
> Hey @gldtn!
> 
> Can you be more specific about what is wrong? Nothing is jumping out at me from your screenshot.

For example, the scopes(words in this case): local, require, @param, if, then, elseif, for, in, do, nvim_set_hl are all in white on my end, vs the screenshot from the readme you provided.

Here I tried replicating it on this shot:
![cyberdream](https://github.com/user-attachments/assets/ff10f317-60fe-44d9-8ad9-52b891ce5a10)

vs yours:
![shot_original](https://github.com/user-attachments/assets/b81b5d31-5eb1-40c5-8e75-813ee94ee7cd)

**Comment by scottmckendry** [comment]
ah! I see. are you using treesitter and have the lua parser installed?

**Comment by gldtn** [comment]
Hmm, seems like there was a couple of things affecting, first my lame attemp to get .blade.php files to be parsed correctly:

After commenting this out, treesitter started parsing correctly.. 
```lua
-- -- FIXME: this is a hack to add blade support
		-- local parser_config = require("nvim-treesitter.parsers").get_parser_configs()
		--
		-- ---@class ParserConfig
		-- parser_config.blade = {
		-- 	install_info = {
		-- 		url = "https://github.com/EmranMR/tree-sitter-blade",
		-- 		files = { "src/parser.c" },
		-- 		branch = "main",
		-- 	},
		-- 	filetype = "blade",
		-- }
		--
		-- vim.filetype.add({
		-- 	pattern = {
		-- 		[".*%.blade%.php"] = "blade",
		-- 	},
		-- })
```

Colors were also different due to me playing with italics:
```lua
overrides = function(colours)
					return {
						-- italics
						Keyword = { italic = true },
						-- Constant = { italic = true },
						Function = { italic = true },
						Identifier = { italic = true },
						-- non-italics
						Unit = { italic = false },
						Error = { italic = false },
						Number = { italic = false },
					}
				end,
```

What is the proper way to add more italics to the theme without overriding the original colors?

**Comment by scottmckendry** [comment]
The overrides function is intended to override the highlight group entirely, so you'd need to give it a colour otherwise it will just default to the value of `fg`. 

So in this case, your config should look like this:
```lua
overrides = function(colours)
    return {
        -- italics
        Keyword = { fg = colours.orange, italic = true},
        Function = { fg = colours.blue, italic = true },
        -- and so on ...
    }
end,
```
That would be the way to do it inside the cyberdream config. However, a more simple approach in your case could be to set the italic property _after_ cyberdream.nvim has already been set up. In that case, the following example should work:

```lua
return {
    {
        "scottmckendry/cyberdream.nvim",
        dev = true,
        lazy = false,
        priority = 1000,
        config = function()
            require("cyberdream").setup({})
            vim.cmd("colorscheme cyberdream")

            -- define italic groups
            local italic_groups = {
                "Keyword",
                "Function",
            }

            -- must appear AFTER the "colorscheme cyberdream" command
            for _, group in ipairs(italic_groups) do
                vim.cmd("hi " .. group .. " gui=italic cterm=italic")
            end
        end,
    },
}
```lua

**Comment by gldtn** [comment]
I do like that second approach, but that seems like if reverts the colors to the default Treesitter/LSP highlights. At least from the quick test I did here. So might have to go through `:hi` and match the ones I want to change. I ran to this issue [here](https://github.com/scottmckendry/cyberdream.nvim/issues/84) and being using fzf to discover all the highlights colors you set, thanks for the help as always!
